### PR TITLE
Add WebExtensions

### PR DESCRIPTION
**Pull request by bfred-it**
https://github.com/bfred-it/Awesome-WebExtensions

WebExtensions is the name Mozilla gave to Chrome extensions that are now being supported by Firefox and Edge: https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Browser_support_for_JavaScript_APIs

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days,**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge,
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Fuse](https://github.com/vinkla/awesome-fuse)
 - [Heroku](https://github.com/ianstormtaylor/awesome-heroku)
 - [Raspberry Pi](https://github.com/thibmaek/awesome-raspberry-pi) - Credit card-sized computer aimed at teaching kids programming, but capable of a lot more.
+- [Web Extensions](https://github.com/bfred-it/Awesome-WebExtensions) - Chrome, Firefox, and Edge extensions.
```
Comment (sindresorhus) [inline readme.md]: Web Extensions => WebExtensions

Needs a better description.

**Comment by sindresorhus** [comment]
Some feedback:

- The list could use more content. I'm sure there's more great content out there. Do some more gathering work.
- I'd like to see some links to community resources: Stack Overflow tag, IRC, chat, forums, communities, etc. Example: https://github.com/sindresorhus/awesome-electron#community
- Be consistent about ending all descriptions in a `.`
- Drop all the `A` / `An` prefixes in the descriptions.
- Some items are missing a description.
- There should be a succinct summary at the top about what WebExtensions are and the word WebExtensions should be linkified to https://developer.mozilla.org/en-US/Add-ons/WebExtensions
- `Web Extensions` => `WebExtensions` (in the repo description too)
- Drop the `Inspired by the awesome list thing.` part. It's moot with the badge.
- Drop the descriptions in the `Contents` section. Those description are better placed in the actual sections, like you've done with some of them already.
- I'd like to see some more resources like blog posts, tutorials, YouTube videos, etc.

**Comment by bfred-it** [comment]
Thanks! This will take a while. Tracking it here:

> - [ ] The list could use more content. I'm sure there's more great content out there. Do some more gathering work.
> - [ ] I'd like to see some links to community resources: Stack Overflow tag, IRC, chat, forums, communities, etc. Example: <a href="https://github.com/sindresorhus/awesome-electron#community">https://github.com/sindresorhus/awesome-electron#community</a>
> - [x] Be consistent about ending all descriptions in a <code>.</code>
> - [x] Drop all the <code>A</code> / <code>An</code> prefixes in the descriptions.
> - [ ] Some items are missing a description.
> - [ ] There should be a succinct summary at the top about what WebExtensions are and the word WebExtensions should be linkified to <a href="https://developer.mozilla.org/en-US/Add-ons/WebExtensions">https://developer.mozilla.org/en-US/Add-ons/WebExtensions</a>
> - [x] <code>Web Extensions</code> =&gt; <code>WebExtensions</code> (in the repo description too)
> - [x] Drop the <code>Inspired by the awesome list thing.</code> part. It's moot with the badge.
> - [x] Drop the descriptions in the <code>Contents</code> section. Those description are better placed in the actual sections, like you've done with some of them already.
> - [ ] I'd like to see some more resources like blog posts, tutorials, YouTube videos, etc.

**Comment by sindresorhus** [comment]
ping :)

**Comment by bfred-it** [comment]
Mostly done, @erikvold has added a lot of links and [the community section is almost there.](https://github.com/bfred-it/Awesome-WebExtensions/pull/11)

**Comment by davisonio** [comment]
Hey any news on this? Community, video and a few other links would make this list 💯!

**Comment by sindresorhus** [comment]
Closing this for now. Do another PR in a couple of months when the list is more mature and follows all the guidelines.

**Pull request closed without merging**